### ISSUE TITLE
IE, Firefox, Opera  - Clicking Menu Item (in 'Mobile') with sub-menu opens the sub-menu prior to taking you to link

### ISSUE THREAD

**Issue by kirkofalltrades** [mrjasonweaver/flexnav#92]
First, thanks for this code!  It's a great idea and it's working well for us on our latest Drupal project, for the most part :)

I'm seeing an issue in IE 10.0.9200.16736, Firefox 16.0.2, Opera 12.16, prior to hitting the desktop breakpoint.

I am using FlexNav.js 1.2.3

I see this occurring on the 'http://jasonweaver.name/lab/flexiblenavigation/' page as well as on my own project site using the noted browsers.

Basically, click a link within the menu that has a submenu and that submenu opens when clicking the link, then proceeds to the link you had clicked.  This is a little disorienting.  My understanding is that the sub menu should appear only when the .touch-button span is clicked.  It functions this way in Chrome and Safari. It also functions this way on my iPhone and Android devices.

Thanks for your hard work!

**Comment by mrjasonweaver** [comment]
@kirkofalltrades Interesting. I'll look into it.
